coderadar-core/src/main/java/io/reflectoring/coderadar/contributor/port/driver/ListContributorsUseCase.java

```diff
@@ -12,9 +12,9 @@
 
   /**
    * @param projectId The project id.
-   * @param command Contains the filename we are looking for.
-   * @return list of contributors that have made changes to the file with the given filename
+   * @param command Contains the filepath/module path we are looking for.
```

Comment:
Since we can get the contributors for any directory in a project and not just module, maybe you can adjust this to `filepath/directory`. (Same goes for other places where you mention modules specifically).

Reply:
Done